### PR TITLE
Xapid886

### PR DESCRIPTION
**Pull request by sramamoorthy73**
Fetch snapshot each time a scope has changes.

### CODE REVIEW

File: changes.go [hunk 1]
```diff
@@ -227,9 +227,7 @@ func (c *pollChangeManager) getChanges(changesUri *url.URL) error {
 	 */
 	if lastSequence != "" &&
 		getChangeStatus(lastSequence, resp.LastSequence) != 1 {
-		return changeServerError{
-			Code: "Ignore change, already have newer changes",
-		}
+		return nil
```
Comment (f1erro) [inline changes.go]: Might be nice to add a debug statement, indicating that we have newer changes

File: changes.go [hunk 2]
```diff
@@ -264,8 +272,8 @@ func (c *pollChangeManager) handleChangeServerError(err error) {
 		log.Debugf("handleChangeServerError: changeManager has been closed")
 		return
 	}
-	if _, ok := err.(changeServerError); ok {
-		log.Info("Detected DDL changes, going to fetch a new snapshot to sync...")
+	if c, ok := err.(changeServerError); ok {
+		log.Debugf("%s. Fetch a new snapshot to sync...", c.Code)
```
Comment (f1erro) [inline changes.go]: I think this should be an Info

File: changes.go (hunk 1 above)
Comment (sramamoorthy73) [inline changes.go]: The debug statement is already printed at the the parent function. That is why its not here.

File: changes.go (hunk 2 above)
Comment (sramamoorthy73) [inline changes.go]: We should actually use Debug, mainly because that is what we have used elsewhere as well.

Comment (theganyo) [inline changes.go]: Agree with debug. This isn't information that you would generally need to see.

**Pull request merged**